pkg/beholder/config.go

```diff
@@ -11,6 +11,7 @@ type Config struct {
 	InsecureConnection       bool
 	CACertFile               string
 	OtelExporterGRPCEndpoint string
+	OtelExporterHTTPEndpoint string
```

Comment:
It seem redundant to have separate constructors and separate config fields. What about either:
- One constructor that checks for which field is set (and ensures both are not set)
- Two constructors that share one field: `OtelExporterEndpoint`

Reply:
Makes sense

Reply:
@jmank88 Addressed here https://github.com/smartcontractkit/chainlink-common/pull/830/commits/ec6684930d600f2d6c8517bd5f955101a4897f6e